ctapipe/io/simteleventsource.py

```diff
@@ -816,6 +837,38 @@ def _parse_simulation_header(self):
         )
         return {obs_id: simulation_config}
 
+    def _fill_scheduling_and_observation_blocks(self):
+        """fill scheduling and observation blocks must be run after the
+        simulation config is filled
+        """
+
+        az, alt = self.file_.header["direction"]
+        obs_id = self.file_.header["run"]
+
+        sb_dict = {
+            obs_id: SchedulingBlockContainer(
+                sb_id=np.uint64(obs_id),  # simulations have no SBs, use OB id
+                sb_type=SchedulingBlockType.OBSERVATION,
+                producer_id="simulation",
+                observing_mode=ObservingMode.UNKNOWN,
+                pointing_mode=PointingMode.DRIFT,
+            )
+        }
+
+        ob_dict = {
+            obs_id: ObservationBlockContainer(
+                obs_id=np.uint64(obs_id),
+                sb_id=np.uint64(obs_id),  # see comment above
+                producer_id="simulation",
+                state=ObservationBlockState.COMPLETED_SUCCEDED,
+                subarray_pointing_lat=alt * u.deg,
+                subarray_pointing_lon=az * u.deg,
+                subarray_pointing_frame=CoordinateFrameType.ALTAZ,
```

Comment:
the simtel file contains timestamps for the run here:

```
t = Time(self.file_.header['time'], format='unix')
```

maybe use that for `{scheduled,actual}_start`?

Reply:
I thought they were basically meaningless, but yes, I can add them.  The duration is at least not something we can use here.